PAL Initial framework (#1)
polkadot-assurance-legion/pal-docs: applications_criteria.md

# PAL Applications Criteria

All applications under PAL are processed based on two sets of criteria: [eligibility criteria](#eligibility-criteria) and [criteria for prioritisation](#criteria-for-prioritisation).

## Eligibility Criteria

In order to be eligible for funding under PAL, every application must cover the complete list of eligibility criteria:

1. The applicant must be a parachain which is producing blocks on the Polkadot network;
2. The application must specify which Substrate pallets or ink! smart contracts would be audited including refs to GitHub and a summary of the total number lines of code;
3. The code must be open source;
4. The code must be production-ready and under development-freeze - no new commits are allowed unless agreed with the auditors (fixes of identified bugs);
5. The Polkadot Audit Fund will cover a maximum of 80% of the auditing costs up to a limit of 18,000 DOT. The applicant parachain must be ready to cover the remaining costs;

> 6. The applicant parachain will receive the funding from the bounty mechanism in DOT. The applicant may need to handle the payment to the auditor directly in stablecoins or fiat.

Reviewer: ```suggestion
6. The applicant parachain will receive the funding from the bounty mechanism in DOT. The applicant may need to handle the payment to the auditor directly in whatever form is required by the auditor
```

## Criteria for Prioritisation

If there are more applications under PAL in a given round than the available funding, the eligible applications will be subjected to the following criteria to determine priority:

1. Count of active wallets;

> 2. Count of open bi-directional HRMP/XCM channels;

Reviewer 2: ```suggestion
2. Count of open and active bi-directional HRMP/XCM channels;
```
